### ISSUE TITLE
[Question] Why didn't TensorBoard record the data  

### ISSUE THREAD

**Issue by Mcairy** [DLR-RM/stable-baselines3#878]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.

### Question

Previously, Tensorboard recorded with EP_LEN_MEAN and EP_REW_MEAN, but after I transferred super-parameters.
No EP LEN MEAN and EP REW MEAN curves

### Additional context

Here are the hyperparameters I tuned
model = SAC('MlpPolicy', normalized_env,learning_rate=7.3e-4, train_freq=8, gradient_steps=8, use_sde=True, verbose=0, tensorboard_log='./log/SAC', device='cuda:0')

### Checklist

- [ ] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) (**required**)
- [ ] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by Miffyli** [comment]
Hey. Could you provide a minimal code to replicate what is happening. If a custom environment is involved, that may also be the cause, but we can not provide too much tech support in that regard.

**Comment by PBerit** [comment]
@Mcairy : This also sometimes happens in my application. Sometimes it helps to reload Tensorboard. As Miffyli pointed out another possible cause of this problem is that there are some bugs in the custom environment when calculating and returning the reward. Maybe you should have a closer look at this.

**Comment by araffin** [comment]
> Could you provide a minimal code to replicate what is happening

closing as the issue template was not filled completely.
